src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -1763,9 +1763,24 @@ private bool OnTouchEventForTrack(object source, TouchEventArgs e)
                     valueIndicatorImage.Show();
                 }
 
+                if (null != sliderSlidingStartedHandler)
+                {
+                    SliderSlidingStartedEventArgs args = new SliderSlidingStartedEventArgs();
+                    args.CurrentValue = curValue;
+                    sliderSlidingStartedHandler(this, args);
+                }
```

Comment:
`?.Invoke(...)` 를 사용하시면 null검사와 event 핸들러 호출을 atomic하게 할 수 있어서 더 간결해지면서도 더 안전해 집니다.

args는 다른곳에서 사용되지 않고 event 핸들러에만 전달되기 때문에 variable로 선언하지 않고 바로 argument로 전달하면 가독성이 더 좋아집니다. 이를 위해 CurrentValue는[ 개체 이니셜라이저 ](https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer) 방법을 통해 초기화 할 수 있습니다.
```suggestion
                sliderSlidingStartedHandler?.Invoke(this, new SliderSlidingStartedEventArgs {
                    CurrentValue = curValue
                });
```

Reply:
리뷰 주셔서 감사합니다. 
의견 주신 것처럼 업데이트하였습니다